src/hct_mis_api/apps/targeting/models.py

```diff
@@ -607,16 +607,17 @@ def get_query(self) -> Q:
 
         collectors_ind_query = Individual.objects.filter(
             pk__in=list(collector_primary_qs),
-            delivery_mechanisms_data__is_valid=True,
         )
         # If argument is Yes
         if argument:
             individuals_with_field_query = collectors_ind_query.filter(
```

Comment:
are Yes and No strings? bool("No") is True @pavlo-mk

Reply:
it's a good question let me double check with @mmaciekk

Reply:
updated this one